pkg/volume/rbd/disk_manager.go

```diff
@@ -40,6 +40,8 @@ type diskManager interface {
 	DetachDisk(disk rbdUnmounter, mntPath string) error
 	// Creates a rbd image
 	CreateImage(provisioner *rbdVolumeProvisioner) (r *v1.RBDVolumeSource, volumeSizeGB int, err error)
```

Comment:
I only see this called in a single place, from `rbdVolumeProvisioner`, right? that should only deal with persistentvolumes... should just change the existing function

Reply:
done